CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to
 - ♻️(frontend) simplify stores #402
 - ✨(frontend) update $css Box props type to add styled components RuleSet #423
 - ✅(CI) trivy continue on error #453
+- 🔨(frontend) encapsulated title to its own component #474
```

Comment:
When you will rebase main, you will see we have a new version, you will have to move this part under unrelease.

Reply:
Done